internal/trace/listener.go

```diff
@@ -54,12 +56,15 @@ func (l *Listener) HandlerStarted(ctx context.Context, msg json.RawMessage) cont
 
 	ctx, _ = contextWithRootTraceContext(ctx, msg, l.mergeXrayTraces)
 
-	tracer.Start(
-		tracer.WithService("aws.lambda"),
-		tracer.WithLambdaMode(true),
-		tracer.WithDebugMode(true),
-		tracer.WithGlobalTag("_dd.origin", "lambda"),
-	)
+	if !tracerInitialized {
+		tracer.Start(
+			tracer.WithService("aws.lambda"),
+			tracer.WithLambdaMode(true),
+			tracer.WithDebugMode(true),
+			tracer.WithGlobalTag("_dd.origin", "lambda"),
+		)
+		tracerInitialized = true
```

Comment:
Is this block (lines 59-66) part of the performance increase? Basically if the tracer is already initialized (warm) then there is no need to do the `trace.Start(...)` stuff?